### ISSUE TITLE
pytorch3d install error

### ISSUE THREAD

**Issue by gauraviiita** [concept-graphs/concept-graphs#6]
Hi @krrish94 
Thank you for being so supportive.

I run the **python scripts/generate_gsa_results.py** successfully. I am running the following command  python **slam/cfslam_pipeline_batch.py** but facing an error.
Please let me know how I can fix it. 

(conceptgraph) user@user:~/Pictures/Gaurav/Robotics/concept-graphs/conceptgraph$ python slam/cfslam_pipeline_batch.py \
>     dataset_root=/home/user/Pictures/Gaurav/Robotics/concept-graphs/Datasets/Replica \
>     dataset_config=/home/user/Pictures/Gaurav/Robotics/concept-graphs/conceptgraph/dataset/dataconfigs/replica/replica.yaml \
>     stride=5 \
>     scene_id=room0 \
>     spatial_sim_type=overlap \
>     mask_conf_threshold=0.95 \
>     match_method=sim_sum \
>     sim_threshold=1.2 \
>     dbscan_eps=0.1 \
>     gsa_variant=none \
>     class_agnostic=True \
>     skip_bg=True \
>     max_bbox_area_ratio=0.5 \
>     save_suffix=overlap_maskconf0.95_simsum1.2_dbscan.1_merge20_masksub \
>     merge_interval=20 \
>     merge_visual_sim_thresh=0.8 \
>     merge_text_sim_thresh=0.8
[2023-10-10 20:37:38,504] [INFO] [real_accelerator.py:110:get_accelerator] Setting ds_accelerator to cuda (auto detect)
Setting image height and width to 680 x 1200
Loaded class colors from  /home/user/Pictures/Gaurav/Robotics/concept-graphs/Datasets/Replica/room0/gsa_classes_none_colors.json
  0%|                                                                                                                                                               | 0/400 [00:00<?, ?it/s]/home/user/anaconda3/envs/conceptgraph/lib/python3.10/site-packages/imageio/plugins/pillow.py:297: UserWarning: Loading 16-bit (uint16) PNG as int32 due to limitations in pillow's PNG decoder. This will be fixed in a future version of pillow which will make this warning dissapear.
  warnings.warn(
  0%|▍                                                                                                                                                      | 1/400 [00:01<10:10,  1.53s/it]
Error executing job with overrides: ['dataset_root=/home/user/Pictures/Gaurav/Robotics/concept-graphs/Datasets/Replica', 'dataset_config=/home/user/Pictures/Gaurav/Robotics/concept-graphs/conceptgraph/dataset/dataconfigs/replica/replica.yaml', 'stride=5', 'scene_id=room0', 'spatial_sim_type=overlap', 'mask_conf_threshold=0.95', 'match_method=sim_sum', 'sim_threshold=1.2', 'dbscan_eps=0.1', 'gsa_variant=none', 'class_agnostic=True', 'skip_bg=True', 'max_bbox_area_ratio=0.5', 'save_suffix=overlap_maskconf0.95_simsum1.2_dbscan.1_merge20_masksub', 'merge_interval=20', 'merge_visual_sim_thresh=0.8', 'merge_text_sim_thresh=0.8']
Traceback (most recent call last):
  File "/home/user/Pictures/Gaurav/Robotics/concept-graphs/conceptgraph/slam/cfslam_pipeline_batch.py", line 270, in main
    spatial_sim = compute_spatial_similarities(cfg, fg_detection_list, objects)
  File "/home/user/Pictures/Gaurav/Robotics/concept-graphs/conceptgraph/slam/mapping.py", line 39, in compute_spatial_similarities
    spatial_sim = compute_overlap_matrix_2set(cfg, objects, detection_list)
  File "/home/user/Pictures/Gaurav/Robotics/concept-graphs/conceptgraph/slam/utils.py", line 293, in compute_overlap_matrix_2set
    iou = compute_3d_iou_accuracte_batch(bbox_map, bbox_new) # (m, n)
  File "/home/user/Pictures/Gaurav/Robotics/concept-graphs/conceptgraph/utils/ious.py", line 165, in compute_3d_iou_accuracte_batch
    import pytorch3d.ops as ops
  File "/home/user/anaconda3/envs/conceptgraph/lib/python3.10/site-packages/pytorch3d/ops/__init__.py", line 7, in <module>
    from .ball_query import ball_query
  File "/home/user/anaconda3/envs/conceptgraph/lib/python3.10/site-packages/pytorch3d/ops/ball_query.py", line 10, in <module>
    from pytorch3d import _C
ImportError: /home/user/anaconda3/envs/conceptgraph/lib/python3.10/site-packages/pytorch3d/_C.cpython-310-x86_64-linux-gnu.so: undefined symbol: _ZN3c106detail14torchCheckFailEPKcS2_jRKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.

**Comment by krrish94** [comment]
This looks like a pytorch3d install error. Can you try running just the `import pytorch3d` statement inside an interpreter to see if that works?

(As the error message above indicates, you're running into this error when `import pytorch3d.ops as ops` is executed)

**Comment by krrish94** [comment]
Updated the issue title to reflect the pytorch3d install issue.

**Comment by gauraviiita** [comment]
Hi @krrish94 
Yes, you are right. 
I had installed pytorch3d wrongly. 

[For installing pytorch3d with CUDA support from Anaconda Cloud, on Linux only, there is a different command. Which is given below.](https://github.com/facebookresearch/pytorch3d/blob/main/INSTALL.md#1-install-with-cuda-support-from-anaconda-cloud-on-linux-only)

**$conda install pytorch3d -c pytorch3d**

I followed it and now it working well.

Thank you very much for your support.
